src/win/util.c

```diff
@@ -484,7 +484,7 @@ uint64_t uv__hrtime(double scale) {
    * performance counter interval, integer math could cause this computation
    * to overflow. Therefore we resort to floating point math.
    */
-  return (uint64_t) ((double) counter.QuadPart * hrtime_interval_ * scale);
+  return (uint64_t) (counter.QuadPart * (uint64_t) scale / hrtime_interval_);
```

Comment:
>If the mantissa is 52 bits, that's still enough precision for to support uv__hrtime.

Only reliably for the first 2 weeks of uptime. After that QPC can return numbers that can't fit in a double without losing precision.